.github/sync-repo-settings.yaml

```
@@ -9,16 +9,17 @@ branchProtectionRules:
   requiredStatusCheckContexts:
     - 'cla/google'
     - 'OwlBot Post Processor'
-    # TODO(tswast): enable once we have tests migrated from google-cloud-bigquery
-    # - 'lint'
+    - 'lint'
+    # TODO: Enable once the docs tests ready.
     # - 'docs'
-    # - 'unit (3.8)'
-    # - 'unit (3.9)'
-    # - 'unit (3.10)'
-    # - 'unit (3.11)'
-    # - 'unit (3.12)'
-    # - 'cover'
-    # - 'Kokoro'
+    - 'unit (3.8)'
+    - 'unit (3.9)'
+    - 'unit (3.10)'
```

We aren't currently testing on 3.9 and 3.10. The hope is that testing on earlier and later Python versions should be sufficent, but if you think we need them, we'll need to also update `owlbot.py`.

```suggestion
    - 'unit (3.7)'
    - 'unit (3.8)'
```

---

Corrected the version here. Thanks for catching that!